translations/en.json

```diff
@@ -956,6 +958,13 @@
     "name-of-token": "The name of your token",
     "need-help": "Need help?",
     "new-to-status": "I’m new to Status",
+    "getting-started-with-status": "Getting started with Status",
+    "getting-started-description": "Something here about what keys are and how great and important they are.\n\nYour options:",
```

Comment:
hmm, I don't think it's beneficial to add these messages which are not ready from design/ documentation team to the translations strings. 
It would be better to just use a plain string message and include a todo in the file to update them with the correct translations strings once ready from the respective teams. 
cc @pedro-et @jorge-campo

Reply:
Hey @J-Son89 , from Documentation, the messages in Onboarding are ready and in the Figma file (yellow cards). I guess this was overlooked (?)

Please, see [this Figma link](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=4395-318718&t=hxLkNt8H2npW87PK-0) and let me know if I can be of any help. 

![Screenshot 2023-05-05 at 12 11 52@2x](https://user-images.githubusercontent.com/62282406/236431732-1690124b-9725-4040-8314-b2e794cd4e8c.png)

Reply:
ah thanks @jorge-campo! @yqrashawn can you update with the finalized copy 👍

Reply:
@yqrashawn, let me know if you need any help with that copy. Happy to help.

> **Note**
> We're aware that the final copy in the yellow cards is longer than the one in the mockups. Users will need to scroll to read the entire text.